Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY: all build rubocop test run console
+
+all: build rubocop test run
+
+ifeq ($(OS),Windows_NT)
+	detectedOS := Windows
+else
+	detectedOS := $(shell sh -c 'uname -s 2>/dev/null || echo not')
+endif
+
+docker-compose := docker-compose
+ifeq ($(detectedOS),Linux)
+	docker-compose := sudo docker-compose
+endif
+
+build:
+	$(docker-compose) build
+
+rubocop:
```

Comment:
Getting this error with the rubocop make command.
```
Makefile:20: recipe for target 'rubocop' failed
make: *** [rubocop] Error 123
```

Reply:
Hm, couldn't reproduce this one. Do you have `xargs`?

Reply:
Yep, the command runs just fine when I paste it into the terminal.  Looking at the make version and the regular command they output the same stuff, maybe rubocop outputs a non-zero exit code when it finds offenses and that causes the error for me.  Shouldn't matter though since it has the same output.